dist/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700;800&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/main.css" />
+  <link rel="stylesheet" href="css/media.css">
+  <title>Folio</title>
+</head>
+<body>
+
+  <!-- Burger content -->
+  <div class="burger-content">
+      <div class="avatar">
+        <img src="img/graph_shapes/shape1_small.png" alt="" class="small-shape avatar-lightning">
+        <img src="img/graph_shapes/shape3_small.png" alt="" class="small-shape  avatar-triple-wave">
+        <img src="img/graph_shapes/shape2.png" alt="" class="small-shape avatar-arrow">
+        <div class="download">
```

Comment:
when I click on .download, I expect that the whole block will trigger an action, not just icon

Reply:
done